lab_3_generate_by_ngrams/main.py

```diff
@@ -236,6 +359,15 @@ def _extract_n_grams(
 
         In case of corrupt input arguments, None is returned
         """
+        if not isinstance(encoded_corpus, tuple) or len(encoded_corpus) == 0:
+            return None
+
+        n_grams = []
+        list_encoded_corpus = list(encoded_corpus)
+        for index in range(len(encoded_corpus) + 1 - self._n_gram_size):
+            n_grams.append(tuple(list_encoded_corpus[index: index + self._n_gram_size]))
```

Comment:
please, save the tuple to a variable. It is dificult to read

Reply:
done